safety:
  battery:
    low_threshold : 30.0
    max_voltage : 100.0
    min_voltage : 0.0
  collision:
    warning_distance : 0.5
    stop_distance : 0.25
  recovery:
    rotation_speed : 0.5